test/python/test_fund_flow_ethereum.py

```diff
@@ -25,11 +26,15 @@ def test_fund_flow_ethereum_max_partner_name_length(client, firmware):
 
     ex.process_transaction(tx_infos, b'\x10\x0f\x9c\x9f\xf0"\x00')
     ex.check_transaction_signature(partner)
-    ex.check_address(LEDGER_SIGNER, right_clicks=5)
+    with ex.check_address(LEDGER_SIGNER):
+        navigator.navigate_until_text_and_compare(NavIns(NavInsID.RIGHT_CLICK),
```

Comment:
For no-argument `NavInsId`, it is no more mandatory to explicit the `NavIns`, you should be fine directly putting `NavInsID.RIGHT_CLICK`.